docs/source/release_notes/release-notes.rst

```diff
@@ -42,32 +42,51 @@ develop
+    *    -
+         -
 
-    *    - |new|
-         - Added a benchmark ``KvsGetRowsBenchmarks`` for benchmarking the KVS ``getRows`` method.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/1770>`__)
+=======
+v0.38.0
+=======
 
-    *    - |fixed|
-         - Creating a postgres table with a long name throws if the truncated name (first sixty characters) is the same as that of a different existing table.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/1729>`__)
+6 Apr 2017
 
-    *    - |devbreak|
-         - The ``atlasdb-remoting`` project was removed. We don't believe this was used anywhere, but if you encounter any problems due to the project having being removed, please contact AtlasDB support.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/1750>`__)
+.. list-table::
+    :widths: 5 40
+    :header-rows: 1
+
+    *    - Type
+         - Change
 
     *    - |improved|
          - The default ``sweepBatchSize`` has been changed from 1000 to 100.
            This has empirically shown to be a better batch size because it puts less stress on the underlying KVS.
            For a full list of tunable sweep parameters and default settings, see :ref:`sweep tunable options <sweep_tunable_parameters>`.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1763>`__)
-       
+
+    *    - |fixed|
+         - Reverted `#1524 <https://github.com/palantir/atlasdb/pull/1524>`__, which caused dependency issues in upstream products.
+           Once we have resolved these issues, we will reintroduce the change, which was originally part of AtlasDB 0.37.0.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/1775>`__)
+
     *    - |fixed|
-         - Fixed broken batching in getting large sets of rows in Cassandra.
+         - Creating a postgres table with a long name now throws a ``RuntimeException`` if the truncated name (first sixty characters) is the same as that of a different existing table.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/1729>`__)
+
+    *    - |fixed|
+         - Fixed a performance regression caused by broken batching in getting large sets of rows in Cassandra.
+           The benchmark, intentionally set up in `#1770 <https://github.com/palantir/atlasdb/pull/1770>`__ to highlight the break, shows a 10x performance improvement.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1764>`__)
 
+    *    - |fixed|
+         - Correctness issue fixed in the ``clean-transactions-range`` CLI. This CLI is responsible for deleting potentially inconsistent transactions in the KVS upon restore from backup.
+           The CLI was not reading the entire ``_transactions`` table, and as a result was missing deleting transactions whose start timestamp was before the backup timestamp and commit timestamp was after the backup timestamp.
```

Comment:
This is wordy - I'd probably change it to "...as a result missed deleting transactions that started before the backup and committed after. This only affects backups that were taken online."